vscode/src/client/analyzerClient.ts

```diff
@@ -50,6 +51,11 @@ export class AnalyzerClient {
     this.analyzerServer.on("exit", (code) => {
       this.outputChannel.appendLine(`Analyzer exited with code ${code}`);
     });
+
+    const enableGenerativeAi = this.config!.get("enableGenerativeAI") as boolean;
+    if (enableGenerativeAi) {
+      this.startGenerativeAIServer();
```

Comment:
We need to spawn the analyzerServer **OR** the kai/rpc-server. Think you should update this `start()` to:

1. start on or the other
2. Move the startGenartiveAIServer() bits here.